test/e2e/prometheus_test.go

```diff
@@ -45,7 +45,9 @@ func TestPrometheusMetrics(t *testing.T) {
 	for service, metric := range expected {
 		t.Run(service, func(t *testing.T) {
 			f.ThanosQuerierClient.WaitForQueryReturn(
-				t, 10*time.Minute, fmt.Sprintf(`count(up{service="%s",namespace="openshift-monitoring"} == 1)`, service),
+				// To avoid making the test wait for more than lookback-delta in case Prometheus
+				// wasn't able to write stale markers (because it was down), reduce the lookup period.
+				t, time.Minute, fmt.Sprintf(`count(last_over_time(up{service="%s",namespace="openshift-monitoring"}[1m]) == 1)`, service),
```

Comment:
Just out of curiosity, the lookback delta default is 5m. Why can we shorten the poll here to a minute?
And to be sure...shorter timeouts are great!

Reply:
Because we're using `last_over_time[1m]`, so it's 1m instead of 5m now.